doc/Usage.md

```diff
@@ -21,6 +21,98 @@ $text = $pdf->getText();
 $text = $pdf->getPages()[0]->getText();
 ```
 
+## Extract text positions
+
+You can extract transformation matrix (indexes 0-3) and x,y position of text objects (indexes 4,5).
+
+```php
+$data = $pdf->getPages()[0]->getDataTm();
+
+Array
+(
+    [0] => Array
+        (
+            [0] => Array
+                (
+                    [0] => 0.999429
+                    [1] => 0
+                    [2] => 0
+                    [3] => 1
+                    [4] => 201.96
+                    [5] => 720.68
+                )
+
+            [1] => Document title
+        )
+
+    [1] => Array
+        (
+            [0] => Array
+                (
+                    [0] => 0.999402
+                    [1] => 0
+                    [2] => 0
+                    [3] => 1
+                    [4] => 70.8
+                    [5] => 673.64
+                )
+
+            [1] => Calibri : Lorem ipsum dolor sit amet, consectetur a
+        )
+)
+```
+
+Additionally font identifier - index for `Document::getFonts()` array (index 2) and font size (index 3) can be added to dataTm with config setting.
```

Comment:
I would adjust this sentence a bit:

> When activated via Config setting (`Config::setDataTmFontInfoHasToBeIncluded(true)`) font identifier (index 2) and font size (index 3) can be added to dataTm.

What do you think? Please refine it if to unclear/faulty.

Reply:
Your suggestion is more clear, i changed only the end 


> can be added to dataTm

to

 > are added to dataTm